src/o365/spfx/commands/project/project-upgrade/model/index.ts

```diff
@@ -16,4 +16,5 @@ export * from './VsCodeLaunchJson';
 export * from './VsCodeSettingsJson';
 export * from './WriteManifestsJson';
 export * from './YoRcJson';
-export * from './ScssFile';
\ No newline at end of file
+export * from './ScssFile';
+export * from './ReportData';
```

Comment:
Why is ReportData a part of the project model?

Reply:
I wanted it to be isolated to avoid impacting the rest of the cli, open to suggestions

Reply:
Just reviewed the code, I'm actually not using the class, removed. thanks!